### ISSUE TITLE
vagrant share not working (could not be found)

### ISSUE THREAD

**Issue by JeanMertz** [mitchellh/vagrant#4828]
In one terminal window I type:

```bash
$ vagrant share --disable-http --ssh --http 3000
# ...
# ==> default: Your Vagrant Share is running! Name: ancient-ocelot-xxxx
# ==> default: You're sharing with SSH access. This means that another user
# ==> default: simply has to run `vagrant connect --ssh ancient-ocelot-xxxx`
# ==> default: to SSH to your Vagrant machine.
```

In the other:

```bash
$ vagrant connect --ssh ancient-ocelot-xxxx
# The share 'ancient-ocelot-xxxx' could not be found!
# Verify that the name you used is correct. If it is indeed correct,
# make sure that you have permissions to view this share.
```

I am unsure what "permissions" I need to access the share. I was under the assumption that having the correct name is enough to at least connect (I know the ssh pasword, but I never get asked for it).

Also tried connecting from a different machine, no-go. Without `--ssh`, no-go.

This is on an *Organization* account, I am logged in with `vagrant login`, and I can see the share being active on https://vagrantcloud.com/shares.

**Comment by sethvargo** [comment]
@JeanMertz is this still an issue on Vagrant 1.7.2?

**Comment by JeanMertz** [comment]
@sethvargo sorry, I switched jobs and no longer have access to the account. I'll close this issue. If I ever run into this again, I know where to find it :smile:

**Comment by sethvargo** [comment]
Well congrats on the new gig! :)

**Comment by JeanMertz** [comment]
Thanks! Just so you know, your [switch](https://sethvargo.com/joining-hashicorp/) triggered me to look at my own situation as well. So indirectly, thank you for my [new gig](https://twitter.com/JeanMertz/status/550704528841269248). :wink:

**Comment by PanosKou** [comment]
Hello,
I am running vagrant 1.6.3, and I have the exact same issue. Has this been fixed on a newer version? The 1.7.2 for example.
Thank you.
